### DISCUSSION TITLE
More Documentation

### DISCUSSION THREAD

**Discussion by jimahlstrom** [softerhardware/CWKeyer#31] [General]
The links to the .hex files in README.md give 404 errors. And in general there really needs to be additional documentation. So I would like write access to README.md. There is no edit button, so I guess I need git push access. I can add to README.md, but the real software authors will need to check if what I write is really correct. Is that OK? Jim, N2ADR

**Comment by softerhardware** [comment]
Hi Jim,
Yes, the README.md is considered a file in the repository. We haven't started the wiki yet. I added you as a collaborator.

73,
Steve
kf7o

**Comment by jimahlstrom** [comment]
I forgot to post this, but I made some modifications to README.md. If they are not correct please let me know.
